src/includes/class-wp-cli.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Network Management Command
+ *
+ */
+class WPMultiNetwork_Command extends WP_CLI_Command {
+
+	/**
+	 * @var array $obj_fields Default fields to display for each object.
+	 */
+	protected $obj_fields = array(
+		"id",
+		"domain",
+		"path"
+	);
+
+	/**
+	 * Add a Network
+	 *
+	 * <domain>
+	 * : Domain for network
+	 *
+	 * <path>
+	 * : Path for network
+	 *
+	 * [--site_name=<site_name>]
+	 * : Name of new network
+	 *
+	 * [--clone_network=<clone_network>]
+	 * : ID of network to clone
+	 *
+	 * [--options_to_clone=<options_to_clone>]
+	 * : Options to clone to new network
+	 *
+	 */
+	public function create( $args, $assoc_args ) {
+
+		list ( $domain, $path ) = $args;
+
+		$defaults   = array( 'site_name' => false, 'clone_network' => false, 'options_to_clone' => false );
+		$assoc_args = wp_parse_args( $assoc_args, $defaults );
+
+		$clone_network = $assoc_args['clone_network'];
+		$options_to_clone = false;
+
+		if ( ! empty( $clone_network ) && !network_exists( $clone_network ) ) {
+			WP_CLI::error( sprintf( 'Clone network %s doesn\'t exist.', $clone_network ) );
```

Comment:
Translations in WP CLI are a little tricky. Not a lot of people are doing translatable string for commands. 

I mean take for example [this](https://github.com/Automattic/Co-Authors-Plus/blob/master/php/class-wp-cli.php#L112) and that is from Automattic.

If I do run the string through an underscore function, do I have regenerate the translation files as well?